src/main/java/com/thebois/views/game/ResourceView.java

```diff
@@ -0,0 +1,69 @@
+package com.thebois.views.game;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+
+import com.badlogic.gdx.graphics.Color;
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.graphics.g2d.Batch;
+
+import com.thebois.models.world.resources.IResource;
+import com.thebois.models.world.resources.ResourceType;
+import com.thebois.views.TextureUtils;
+
+/**
+ * View that renders the resources that exist in the world.
+ */
+public class ResourceView implements IView {
+
+    private static final Color WATER_COLOR = Color.valueOf("#1E90FF");
+    private static final Color TREE_COLOR = Color.valueOf("#006400");
+    private static final Color ROCK_COLOR = Color.valueOf("#808080");
```

Comment:
With more resources in the future this will not look great, any way we can store this somewhere else in view and then fetch it? (Obv not in model to be clear)

Reply:
Where would you suggest doing this then? I feel like this a view related thing no?